src/general/semidiscretization.jl

```diff
@@ -605,81 +608,98 @@ end
 
 # NHS updates
 # To prevent hard to spot errors there is not default version
-
```

Comment:
can you change the comment above to e.g.
`To avoid hard-to-find bugs, there is no default version`

Also, since there is no default version, I'm wondering if it might make sense to comment above the updates and classify them - something like "normal update", "special update only when neighbor is moving" and so on...